test/types.string.test.js

```diff
@@ -0,0 +1,29 @@
+'use strict';
+
+/**
+ * Module dependencies.
+ */
+
+const mongoose = require('./common').mongoose;
+
+const assert = require('assert');
+
+/**
+ * Test.
+ */
+
+describe('types.string', function() {
+  describe('Schema.Types.String.setDefaultOptions(...)', function() {
+    it('when given an option, sets it', () => {
+      // Arrange
+      const mongooseInstance = new mongoose.Mongoose();
+
+      // Act
+      mongooseInstance.Schema.Types.String.setDefaultOption('trim',true);
```

Comment:
I'd recommend just `set()` rather than `setDefaultOption()`. That's more consistent with [`mongoose.set()`](https://mongoosejs.com/docs/api/mongoose.html#mongoose_Mongoose-set)

Reply:
I agree, `set()` sounds better, consistency-wise.